package.json

```diff
@@ -7,6 +7,9 @@
   "author": "Fl_GUI",
   "license": "MIT",
   "private": false,
+  "prettier": {
+    "semi": false
```

Comment:
this is me being lazy :sweat_smile: I'd rather have semicolons where needed.

Reply:
Sure. Personally I prefer `false` here as it feels cleaner, but I'll flip it to `true` and lint again.